cmds/coredhcp/config.yml.example

```diff
@@ -10,4 +10,8 @@ server4:
     listen: '0.0.0.0:67'
     interface: "eth0"
     plugins:
-        - server_id: 192.168.1.12
+        # - server_id: 10.10.10.1
+        # - dns: 8.8.8.8 8.8.4.4
+        # - router: 10.10.10.1
+        # - netmask: 255.255.255.0
+        # - file: leases.txt
```

Comment:
Lets add the range plugin you will be using in the example. That is the one you will be using right?